File: DotNet/DataAndReportExportSample/DataAndReportMethods.cs
using System;
using System.IO;
using System.Collections.Generic;
using System.Globalization;
using System.Threading.Tasks;
using System.Collections;
using System.Reflection;
using Newtonsoft.Json.Linq;
using System.Linq;

using RESTAPISDK.Standard;
using RESTAPISDK.Standard.Models;
using RESTAPISDK.Standard.Controllers;
using RESTAPISDK.Standard.Utilities;
using RESTAPISDK.Standard.Exceptions;


namespace DataAndReportExport
{
    class DataAndReportMethods : ControllerBase
    {
        public async Task<object> GetLiveboardData(RESTAPISDKClient p_client, string p_id, string p_vizId)
        {
            var body = new TspublicRestV2DataLiveboardRequest();
            body.Id = p_id;
            body.FormatType = FormatType1Enum.FULL;

            if (p_vizId != null)
            {
                body.VizId = new List<string>();
                body.VizId.Add(p_vizId.ToString());
            }

            DataController dataController = p_client.DataController;
            object result = await dataController.LiveboardDataAsync(body);

            return result;
        }
    }
}

File: Python/DataAndReportExportSample/DataAndReportExport/DataAndReportMethods.py
#! /usr/bin/env python3
# pylint: disable= no-name-in-module
from restapisdk.models.liveboard_data_format_type_enum \
    import LiveboardDataFormatTypeEnum
from restapisdk.models.tspublic_rest_v_2_data_liveboard_request \
    import TspublicRestV2DataLiveboardRequest

from Python.DataAndReportExportSample.DataAndReportExport.ControllerBase\
    import ControllerBase


class DataAndReportExport(ControllerBase):
    def GetLiveboardData(self, client, p_Id, viz_Id):
        dataController = client.data
        body = TspublicRestV2DataLiveboardRequest()
        body.id = p_Id
        body.format_type = LiveboardDataFormatTypeEnum.FULL
        if viz_Id is not None:
            body.viz_id = [viz_Id]

        result = dataController.liveboard_data(body)
        return result
